### PR TITLE
Add support for Set to ConstantReader

### PR DESCRIPTION
**Pull request by kevmoo**
Added two new members, updated two existing members

### CODE REVIEW

File: source_gen/lib/src/constants/reader.dart [hunk 1]
```diff
@@ -112,6 +112,12 @@ abstract class ConstantReader {
   /// Constant as a `List` value.
   List<DartObject> get listValue;
 
+  /// Whether this constant represents a `List` value.
```
Comment (jakemac53) [inline source_gen/lib/src/constants/reader.dart:115]:
```suggestion
  /// Whether this constant represents a `Set` value.
```

File: source_gen/lib/src/constants/reader.dart [hunk 2]
```diff
@@ -112,6 +112,12 @@ abstract class ConstantReader {
...
+  bool get isSet => false;
+
+  /// Constant as a `List` value.
```
Comment (jakemac53) [inline source_gen/lib/src/constants/reader.dart:118]:
```suggestion
  /// Constant as a `Set` value.
```

**Comment by kevmoo** [comment]
Thanks for the fixes!!

**Comment by kevmoo** [comment]
I borked the history. I feel shame. I disabled merge commits. 🤦

**Comment by jakemac53** [comment]
lol no squash/merge was the correct option

**Comment by jakemac53** [comment]
(it does actually still credit me on the one commit, if I cared, lol)

**Comment by jakemac53** [comment]
or wait lol you didn't squash, whatever though

**Pull request merged**